app/src/main/java/com/cornellappdev/score/components/SportCard.kt

```diff
@@ -0,0 +1,137 @@
+package com.cornellappdev.score.components
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Card
+import androidx.compose.material3.CardDefaults
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.Alignment
+import com.example.score.R
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.cornellappdev.score.theme.Style
+import com.cornellappdev.score.theme.Style.dateText
+import com.cornellappdev.score.theme.Style.teamName
+import com.cornellappdev.score.theme.Style.universityText
+
+@Composable
+fun SportCard(
+    teamLogo: Painter,
+    team: String,
+    date: String,
+    location: String,
+    sportName: String,
+    sportIcon: Painter
+) {
+    Card(
+        modifier = Modifier
+            .padding(16.dp)
+            .fillMaxWidth(),
+        colors = CardDefaults.cardColors(containerColor = Color.White)
```

Comment:
One tip to boost your component game:

Take in a `modifier: Modifier = Modifier` as input to the component, and instead of doing `modifier = Modifier` on the `Card`, work off of the `modifier` you take in as argument. This allows much more customization of the component by people using it.

Also, I think you should take off the `.padding(16.dp)` here. People calling the `SportCard()` can then use that `modifier` argument to add padding themselves if needed. The fill max width is ok to keep.